### PR TITLE
Feature/presentation details fix

### PR DESCRIPTION
**Pull request by Jsyro**
Data that is requested using the singular 'name' field are put elsewhere in the presentation as well. here is how that looks on screen

image

this is a preview of that discovery, but this FE component can handle the first two columns now. 

image

There are many more UI and code changes to be made once a UX design is made

### CODE REVIEW

**Comment by Jsyro** [comment]
Added raw json viewer for kyle. 

image

File: services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue [hunk 1]
```diff
@@ -1,40 +1,84 @@
...
+            presentation.acapy.presentation_exchange.presentation_request
+              .requested_attributes[attr_name].name
+          }}</b>
+          : {{ val.raw }}
+        </li>
+        <!-- requested_attribute using 'name' string w/o restrictions -> revealed self-attested values -->
+        <!-- requested_predicates -> unrevealed attributes -->
+      </div>
     </ul>
+    <Accordion class="qr-accordion">
```
Comment (loneil) [inline services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue:55]: Class `qr-accordion` seems to be from the QR Code component? Unused here anyways it appears

File: services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue [hunk 2]
```diff
@@ -1,40 +1,84 @@
...
+          v-for="(val, attr_name, i) in presentation.acapy.presentation_exchange
+            .presentation.requested_proof.revealed_attrs"
           :key="i"
           class="presentation-attr-value"
         >
-          <b>{{ attr_name }}</b> : {{ val.raw }}
-          <br />
-        </span>
-      </li>
+          <b>{{
```
Comment (loneil) [inline services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue:45]: Generally use `<strong>` over bold tag in html5 these days
https://html.com/tags/b/

**Review by loneil — APPROVED** [review]
Minor code style comments

Tried locally with Kyle's tenant, looks great, love the raw content viewer

**Pull request merged**